### PR TITLE
Bug: User's comments on their tracks should not display on homepage 

### LINKED ISSUE #460: Comments on home when logged in / moderator differ from guest
Please look at the different between the home and https://alonetone.com/comments when logged out.

### PR DESCRIPTION
**Pull request by innatewonderer**
Fixes #460 

## "Ready For Review" Checklist

* [x] PR title accurately summarizes changes
* [X] PR description
* [X] Does this fix any open issues?
    * Issue #460 
    * The scope `to_other_members ` should filter out comments where `user.id == commenter.id`  OR if comment's `remote_ip` address matches owner's `current_login_ip `
    * Admin will still see all comments on homepage.
* [X] What changes are you making?
Fix existing bug, where comments were not displaying on homepage accurately. Only comments by users to other users should be displayed.
Cleaning up assets/comments and user assets to be able to easily read comments on Percy reports.
* [X] Why did you implement it this way?
Scope existed already, just needed to fix the reference.
* [X] Is there anything reviewers need to know or pay attention to?
The scope itself is a bit confusing. 
* [X] Does anything special need to happen for deployment?
No
* [ ] ~Optional: Inline github PR comments explaining context of tricky / complicated / unexpected lines so reviewers can follow along.~
* [X] If appropriate, new tests were added for isolated methods or new endpoints
* [X] All tests green
* [x] Percy changes are purposeful or [explained](https://percy.io/alonetone/alonetone/builds/2079146?utm_campaign=alonetone&utm_content=alonetone&utm_source=github_status_public)
The first two pages' changes are due to new fixtures being added.
Fullscreen_6_23_19__12_48_AM

* [x] I booted up the branch locally, exercised any new code I wrote.
* [ ] ~If css was touched, I verified changes are happy on mobile (via Percy is ok)~
* [ ] ~I opened an issue for any logical followups~

### CODE REVIEW

**Comment by innatewonderer** [comment]
Found it.
It's because I made them all comments to `commentable: valid_mp3 (Asset)`
I'll create a new asset so that we don't pollute the pool.

**Comment by innatewonderer** [comment]
These Percy conflicts were actually legit (due to new asset and its comments)
So I approved them.
Ready for your blessing and merge.

**Comment by innatewonderer** [comment]
Updated fixture for comment (I confused user vs commenter and gave user incorrect asset). Should be good to go now @sudara

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def index
       set_comments_made
     else
       @page_title = "Recent Comments"
-      @pagy, @comments = pagy(Comment.on_track.includes(commenter: :pic, commentable: { user: :pic }).public_or_private(moderator?))
+      @pagy, @comments = pagy(Comment.includes(commenter: :pic, commentable: { user: :pic }).public_or_private(moderator?))
```
Comment (sudara) [inline app/controllers/comments_controller.rb]: We used to have comments on multiple other types of records. So removing `on_track` will potentially surface some errors on older data.

**Comment by innatewonderer** [comment]
Approved the changes in Percy (new asset that I added to .yml file is apparently `hot` on the charts of alonetone :) so it pushed the other records down.

**Comment by innatewonderer** [comment]
I think Percy is complaining because I approved a change in https://github.com/sudara/alonetone/pull/538 😭 Can't win with him :/

**Comment by sudara** [comment]
Renamed the title to be clear about primary purpose of the branch.

**Pull request merged**